### PR TITLE
Acrolinx quality improvements: auditpol-remove.md

### PR DESCRIPTION
**Pull request by gswashington**

### CODE REVIEW

**Comment by Jak-MS** [comment]
#assign: @robinharwood, @Xelu86 

Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/administration/windows-commands/auditpol-remove.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
+Applies to: Windows Server 2022, Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
 
-Removes the per-user audit policy for a specified account or all accounts.
+The auditpol remove command removes the per-user audit policy for a specified account or all accounts.
 
-To perform *remove* operations on the *per-user* policy, you must have **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *remove* operations if you have the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows additional access that is not necessary to perform the overall *remove* operations.
+To perform *remove* operations on the *per-user* policy, you must have **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *remove* operations if you have the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows additional access that isn't necessary to perform the overall *remove* operations.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/windows-commands/auditpol-remove.md:15]:
```suggestion
To perform *remove* operations on the *per-user* policy, you must have **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *remove* operations if you've the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows more access than is necessary to perform the overall *remove* operations.
```

**Review by robinharwood — CHANGES_REQUESTED** [review]
Hey @gswashington - could you also look to replace the word additional on line 52, for example, Other references.

Thanks,

Robin

**Comment by robinharwood** [comment]
@gswashington - A couple of minor changes. Merging now.

**Review by robinharwood — APPROVED** [review]

**Pull request merged**